javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -584,24 +580,16 @@ private Vector(BitMappedTrie<T> trie) {
     }
 
     @Override
-    public Vector<Vector<T>> combinations() {
-        return rangeClosed(0, length()).map(this::combinations).flatMap(Function.identity());
-    }
+    public Vector<Vector<T>> combinations() { return rangeClosed(0, length()).map(this::combinations).flatMap(Function.identity()); }
```

Comment:
Just a question :) Did you change your local formatter? Now we have many single-line methods. It does not really hurt but I think you may have a reason to reformat. Generally ok for me!